packages/pouchdb-ajax/src/request-browser.js

```diff
@@ -126,6 +153,7 @@ function xhRequest(options, callback) {
   try {
     xhr.open(options.method, options.url);
   } catch (exception) {
+    cleanUp();
```

Comment:
We shouldnt need to do any cleanup here, the `setTimeout` has not been called yet